### ISSUE TITLE
Request to shutdown Anon-SMS

### ISSUE THREAD

**Issue by 0n1cOn3** [HACK3RY2J/Anon-SMS#57]
Hi

We getting the information that you have "stolen" the code from the Original Developer [SpeedX](https://github.com/thespeedx) and his Project [TBomb](https://github.com/thespeedx/tbomb). 
As per [licence](https://github.com/TheSpeedX/TBomb/blob/master/LICENSE) its prohibited to modifiy the source code and publishing without having the agreement from the developer. 

Here a snippet from the GNU GENERAL PUBLIC LICENSE : 
  
   **5. Conveying Modified Source Versions.**

    You may convey a work based on the Program, or the modifications to
    produce it from the Program, in the form of source code under the
    terms of section 4, provided that you also meet all of these conditions:

    a) The work must carry prominent notices stating that you modified
    it, and giving a relevant date.

    b) The work must carry prominent notices stating that it is
    released under this License and any conditions added under section
    7.  This requirement modifies the requirement in section 4 to
    "keep intact all notices".

    c) You must license the entire work, as a whole, under this
    License to anyone who comes into possession of a copy.  This
    License will therefore apply, along with any applicable section 7
    additional terms, to the whole of the work, and all its parts,
    regardless of how they are packaged.  This License gives no
    permission to license the work in any other way, but it does not
    invalidate such permission if you have separately received it._

If you do not comply with this request, we will be forced to make a DMCA takedown request with SpeedX.

Best thanks
h@x

**Comment by HACK3RY2J** [comment]
I don't believe to have the source code copied, the model has been used for reference and maybe it may be similar in case of coding.

Kindly cross check the source code......

**Comment by 0n1cOn3** [comment]
I have read the source code twice. And has been slightly modified and thats it. The main code is stolen as per license given.
